python/ray/tune/integration/wandb.py

```diff
@@ -19,6 +19,19 @@
 _WANDB_QUEUE_END = (None, )
 
 
+def _clean_config(config):
+    # Fixes https://github.com/ray-project/ray/issues/10631
+    new_config = {}
+    for k, v in config.items():
+        try:
+            pickle.dumps(v)
+            new_config[k] = v
+        except Exception:
+            # give up, similar to _SafeFallBackEncoder
+            new_config[k] = str(v)
+    return new_config
```

Comment:
I think it would be good to handle nested configs here. Right now this would discard a whole sub dictionary if one if its subkeys fails to pickle.

Also relating to #10426, should we maybe filter for numbers and strings explicitly (including dicts and lists of numbers/strings) and just discard the rest?